.github/ISSUE_TEMPLATE/release-issue.md

```diff
@@ -0,0 +1,75 @@
+---
+name: Release umbrella issue
+about: Release Issue
+title: 'Mobile release '
+labels: release
+assignees: ''
+
+---
+Issue to manage mobile release <version>
+
+### Scope
+[comment]: # (Add milestone from github)
+https://github.com/status-im/status-react/milestone/
+
+- release notes: 
+- release branch: 
+
+
+### Testing info 
+[comment]: # (TestRail info, added by QA)
+Main test suite: 
+Upgrade (e2e): 
+
+### Builds in TestFlight and PlayStore to push as a release
+[comment]: # (Will be added by QA after testing approving of build)
+
+### Process
+#### 1) Kick-off
+- [ ] 1. Translation PRs is merged before cutting branch (Jinho)
+- [ ] 2. Prepare scope of visible changes based on commits history, started from previous release branch cut ([example]( https://notes.status.im/1.14-release-notes?view))
+- [ ] 3. Update https://github.com/status-im/status-react/blob/develop/RELEASES.md with release notes (make PR based on previous step)
+- [ ] 4. Create 1.14.0 release branch and get successful builds (doc is [here](https://github.com/status-im/status-react/blob/develop/doc/RELEASE_GUIDE.md) so, testing can be started)
+- [ ] 5. Based on release scope, ask for comms (Jonny) 
+- [ ] 6. Make sure that assets (screenshots, video) in stores are up to date. If should be updated, ask @cammellos  for help
+
+#### 2) Testing
+[comment]: # (Section will be maintained by QA)
+- [ ] Dogfooding (add notes, if it was held) 
+- [ ] Regression testing for release candidate build
+- [ ] Upgrade tests
+- [ ] Fix known blocker(s) found during regression testing
+
+##### Blockers
+[comment]: # (Section for blocker issues; should be fixed and merged by release-manager. Remove if no blockers found)
+- [ ] 1 blocker:
+
+=============================================================================================
+
+- [ ] Cherry-pick fix(es) to the release branch
+- [ ] Verify fix(es) on new release candidate build
+- [ ] Retest
+- [ ] Keep release build forever (button in Jenkins)
+
+#### 3) Submission
+
+- [ ] 1. Release notes written and added (500 character limit for updates (Android)
```

Comment:
Maybe we should make it clear that we are also talking about `RELEASES.md` in the repo.

Reply:
should be resolved; added om this stage that PR should be ready to merge and on the last stage that it should be merged.

Thank you for thorough review.